We got a not be fully compatible error on build

Reproduced steps: 
* pull the code base
* dotnet build

output

```
Microsoft (R) Build Engine version 17.0.0-preview-21460-01+8f208e609 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
/home/aloisdg/Source/Cardidy/src/Tests/Tests.csproj : warning NU1701: Package 'Dedge.Cardizer 0.5.0-alpha' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project. [/home/aloisdg/Source/Cardidy/Cardidy.sln]
/home/aloisdg/Source/Cardidy/src/Tests/Tests.csproj : warning NU1701: Package 'NUnit3TestAdapter 4.0.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project. [/home/aloisdg/Source/Cardidy/Cardidy.sln]
  All projects are up-to-date for restore.
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
/home/aloisdg/Source/Cardidy/src/Tests/Tests.csproj : warning NU1701: Package 'Dedge.Cardizer 0.5.0-alpha' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
/home/aloisdg/Source/Cardidy/src/Tests/Tests.csproj : warning NU1701: Package 'NUnit3TestAdapter 4.0.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
  Dedge.Cardidy -> /home/aloisdg/Source/Cardidy/src/Dedge.Cardidy/bin/Debug/netstandard2.0/Dedge.Cardidy.dll
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
/home/aloisdg/Source/Cardidy/src/Tests/Tests.csproj : warning NU1701: Package 'Dedge.Cardizer 0.5.0-alpha' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
/home/aloisdg/Source/Cardidy/src/Tests/Tests.csproj : warning NU1701: Package 'NUnit3TestAdapter 4.0.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
  You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
  Tests -> /home/aloisdg/Source/Cardidy/src/Tests/bin/Debug/netstandard2.0/Tests.dll
  Tests -> /home/aloisdg/Source/Cardidy/src/Tests/bin/Debug/net6.0/Tests.dll
  App -> /home/aloisdg/Source/Cardidy/src/App/bin/Debug/net6.0/App.dll

Build succeeded.

/home/aloisdg/Source/Cardidy/src/Tests/Tests.csproj : warning NU1701: Package 'Dedge.Cardizer 0.5.0-alpha' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project. [/home/aloisdg/Source/Cardidy/Cardidy.sln]
/home/aloisdg/Source/Cardidy/src/Tests/Tests.csproj : warning NU1701: Package 'NUnit3TestAdapter 4.0.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project. [/home/aloisdg/Source/Cardidy/Cardidy.sln]
/home/aloisdg/Source/Cardidy/src/Tests/Tests.csproj : warning NU1701: Package 'Dedge.Cardizer 0.5.0-alpha' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
/home/aloisdg/Source/Cardidy/src/Tests/Tests.csproj : warning NU1701: Package 'NUnit3TestAdapter 4.0.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
/home/aloisdg/Source/Cardidy/src/Tests/Tests.csproj : warning NU1701: Package 'Dedge.Cardizer 0.5.0-alpha' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
/home/aloisdg/Source/Cardidy/src/Tests/Tests.csproj : warning NU1701: Package 'NUnit3TestAdapter 4.0.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
    6 Warning(s)
    0 Error(s)

Time Elapsed 00:00:01.04
```

I think this may be linked to #45

--- a/src/Tests/Tests.csproj
+++ b/src/Tests/Tests.csproj
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
 	  <LangVersion>10.0</LangVersion>
 	  <Nullable>enable</Nullable>
 

